sources/specUtils.ts

```diff
@@ -123,6 +125,27 @@ export async function loadSpec(initialCwd: string): Promise<LoadSpecResult> {
   if (typeof rawPmSpec === `undefined`)
     return {type: `NoSpec`, target: selection.manifestPath};
 
+  const pathToListFile = path.join(getInstallFolder(), `packageJsonList.json`);
+  try {
+    const file = await fs.promises.open(pathToListFile, `r+`);
+    const list = JSON.parse(await file.readFile(`utf8`)) as Array<string>;
+    if (!list?.includes(selection.manifestPath)) {
+      list.push(selection.manifestPath);
+      await file.truncate(0);
+      await file.write(JSON.stringify(list), 0);
+    }
+    await file.close();
+  } catch (err) {
+    if ((err as NodeError)?.code === `ENOENT`) {
+      // If the file doesn't exist yet, we can create it.
+      await fs.promises.writeFile(pathToListFile, JSON.stringify([selection.manifestPath]))
+        .catch(debugUtils.log);
+    } else {
+      // In case of another error, ignore it.
+      debugUtils.log(`Failed to update packageJsonList file because of the following error: ${err}`);
+    }
+  }
+
```

Comment:
Do we still need that ?

Reply:
I guess no matter if we want to implement this, it doesn't belong to this PR. Let me remove it for now.
```suggestion
```